src/utils.ts

```diff
@@ -16,32 +33,137 @@ export const generateTSSEndpoints = (tssNodeEndpoints: string[], parties: number
 };
 
 export function storageAvailable(type: string): boolean {
-  let storageExists = false;
-  let storageLength = 0;
   let storage: Storage;
   try {
-    storage = window[type];
-    storageExists = true;
-    storageLength = storage.length;
+    if (type === "localStorage") storage = window.localStorage;
+    else storage = window.sessionStorage;
+
     const x = "__storage_test__";
     storage.setItem(x, x);
     storage.removeItem(x);
+
     return true;
   } catch (error) {
-    return (
-      error &&
-      // everything except Firefox
-      (error.code === 22 ||
-        // Firefox
-        error.code === 1014 ||
-        // test name field too, because code might not be present
-        // everything except Firefox
-        error.name === "QuotaExceededError" ||
-        // Firefox
-        error.name === "NS_ERROR_DOM_QUOTA_REACHED") &&
-      // acknowledge QuotaExceededError only if there's something already stored
-      storageExists &&
-      storageLength !== 0
-    );
+    return false;
   }
 }
+
+// TODO think which conversion functions to keep and how to export them.
+
+/**
+ * Parses a JWT Token, without verifying the signature.
+ * @param token - JWT Token
+ * @returns Extracted JSON payload from the token
+ */
+export function parseToken(token: string) {
+  const base64Url = token.split(".")[1];
+  const base64 = base64Url.replace("-", "+").replace("_", "/");
+  return JSON.parse(window.atob(base64 || ""));
+}
+
+/**
+ * Converts a mnemonic to a factor key.
+ * @param tKey - An initialized tKey instance.
+ * @param shareMnemonic - The mnemonic to convert.
+ * @returns The factor key.
+ */
+export async function mnemonicToKey(tKey: ThresholdKey, shareMnemonic: string): Promise<BN> {
+  const factorKey = await (tKey.modules.shareSerialization as ShareSerializationModule).deserialize(shareMnemonic, "mnemonic");
+  return factorKey;
+}
+
+/**
+ * Converts an arbitrary string to a factor key over an elliptic curve.
+ * @param ec - The elliptic curve.
+ * @param shareMnemonic - The mnemonic to convert.
```

Comment:
there params descriptions are not matching with actual function params and why this function is required since we already have one above

Reply:
yeah removing this function